pkg/resolver/search_test.go

```diff
@@ -12,12 +12,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_SearchResolver_Count(t *testing.T) {
+var PropTypes map[string]string
 
+func Test_SearchResolver_Count(t *testing.T) {
+	PropTypes := make(map[string]string)
```

Comment:
- Do we need to declare outside of the test?
- Maybe `propTypeMock` is a better name and would avoid confusion. 
```suggestion

func Test_SearchResolver_Count(t *testing.T) {
	propTypesMock := map[string]string {"kind":"string"}
```